content/product/pulumi-insights.md

```diff
@@ -8,43 +8,65 @@ aliases:
 - /insights
 
 overview:
-    titleTop: Intelligent Cloud Management
-    titleBottom: 
+    header: Intelligent Cloud Management
+    title: A central hub to securely manage all of your environments, secrets, and configurations
+    body: |
+      - **Stop secret sprawl.** Pull and sync secrets and configuration with any secrets store – HashiCorp Vault, AWS Secrets Manager, Azure Key Vault, GCP Secret Manager, 1Password, and more – and consume in any application, tool, or CI/CD platform.  
+      - **Trust (and prove) your secrets are secure.** Adopt dynamic, short-lived secrets on demand as a best practice. Lock down every environment with RBAC, versioning, and a full audit log of all changes.  
+      - **Ditch `.env` files.** No more copying-and-pasting secrets or storing them in plaintext on dev computers. Developers can easily access secrets via CLI, API, Kubernetes operator, the Pulumi Cloud UI, and SDKs.
+      - **Use with or without Pulumi IaC.** Use Pulumi ESC independently, or use with Pulumi IaC to support storing secrets in config in a more secure way than using plaintext.
     description: |
         Pulumi Insights is an asset and compliance management service that gives search, compliance remediation, resource visualizations, and AI insights over any infrastructure, including resources not provisioned by Pulumi IaC such as AWS CloudFormation, Microsoft ARM, HashiCorp Terraform, or even manually.
     items:
-        - title: Gain Unique Insights
-          icon: eye
+        - title: Security
+          icon: shield
           icon_color: purple
           description: |
             Analyze and dig into your organization’s cloud usage and trends.
 
-        - title: Increase Productivity
-          icon: nodes
+        - title: Compliance
+          icon: gavel
           icon_color: yellow
           description: |
             Reduce lead time from ideation to delivery through Pulumi AI and Pulumi Copilot.
 
-        - title: Improve Operations
-          icon: shield
+        - title: Efficiency
+          icon: cycle
           icon_color: blue
           description: |
             Analyze and update infrastructure to optimize costs, enhance security, and ensure compliance. 
 
+workflow:
```

Comment:
I think there needs to be some text right under Workflow. Should say something like

"Making sense of the millions of cloud resources and resource updates per month, across hundreds of clouds, regions, and accounts is hard. Pulumi Insights provides you with the tools to to navigate all four phases of infrastructure management"